### PR TITLE
Add accessibility conformity mention

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: src/translations/en/accessibility.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Compliance status
+
+**Open Terms Archive** is **not compliant** with the RGAA. The site has not yet been audited.
+
+## Improvement and contact
+
+If you are unable to access a content or service, you can contact the Open Terms Archive manager to be directed to an accessible alternative or to obtain the content in another form.
+
+- E-mail: [contact@opentermsarchive.org](mailto:contact@opentermsarchive.orgcontact@opentermsarchive.orgcontact@opentermsarchive.org)
```
Comment (MattiSG) [inline src/translations/en/accessibility.mdx:17]:
```suggestion
- E-mail: [contact@opentermsarchive.org](mailto:contact@opentermsarchive.org)
```

File: src/translations/en/accessibility.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+We try to respond within 2 business days.
+
+## Recourse
+
+This procedure is to be used in the following case: you have notified the website manager of a lack of accessibility that prevents you from accessing a content or service of the portal and you have not received a satisfactory response.
+
+You can :
+
+- Write a message to the [Human Rights Defender](https://formulaire.defenseurdesdroits.fr/)
+- Contact [the representative of the Defender of Rights in your region](https://www.defenseurdesdroits.fr/saisir/delegues)
```
Comment (MattiSG) [inline src/translations/en/accessibility.mdx:29]:
```suggestion
- Contact [the representative of the Défenseur des droits in your region](https://www.defenseurdesdroits.fr/saisir/delegues)
```

File: src/translations/fr/accessibility.mdx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+## État de conformité
+
+**Open Terms Archive** est **non conforme** avec le RGAA. Le site n’a encore pas été audité.
+
+## Amélioration et contact
+
+Si vous n’arrivez pas à accéder à un contenu ou à un service, vous pouvez contacter le responsable de Open Terms Archive pour être orienté vers une alternative accessible ou obtenir le contenu sous une autre forme.
+
+- E-mail : [contact@opentermsarchive.org](mailto:contact@opentermsarchive.orgcontact@opentermsarchive.orgcontact@opentermsarchive.org)
+- Twitter : @openTerms
```
Comment (MattiSG) [inline src/translations/fr/accessibility.mdx:18]: Same feedback as for the English version.

**Review by MattiSG — CHANGES_REQUESTED** [review]
Please take a bit of time to double-check content before opening the PR 🙂 The triple email address should have been easy to spot 😉

File: src/translations/en/accessibility.mdx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## Recourse
+
+This procedure is to be used in the following case: you have notified the website manager of a lack of accessibility that prevents you from accessing a content or service of the portal and you have not received a satisfactory response.
+
+You can:
+
+- Write a message to the [Défenseur des droits](https://formulaire.defenseurdesdroits.fr/)
+- Contact [the representative of the Défenseur des droits in your region](https://www.defenseurdesdroits.fr/saisir/delegues)
+- Send a letter by post (free of charge, do not put a stamp):
```
Comment (MattiSG) [inline src/translations/en/accessibility.mdx:30]:
```suggestion
- Send a letter by post (free of charge in France, do not put a stamp):
```

**Review by MattiSG — APPROVED** [review]
Tested locally 👍 Thanks @clementbiron 🙂

**Review by martinratinaud — APPROVED** [review]

**Comment by martinratinaud** [comment]
Also, I believe it would be great, if we want to ease the creation of pages, to have a "catch-all" route in `pages` (`[...slug].tsx`) that will:

- check if the corresponding `mdx` file exists
- if no -> return 404
- if yes -> display the content and use `frontMatter` to add SEO data directly within the mdx file

I believe this would end up with something similar than what github pages provide

Let me know if this is needed soon or if, considering the few pages we need right now, is overkill.

**Comment by clementbiron** [comment]
@martinratinaud thanks Martin, let's talk about your suggestion's at the next planning.

**Pull request merged**